client/dashboard/utils.js

```diff
@@ -49,14 +49,78 @@ export function getProductIdsForCart(
 	profileItems,
 	includeInstalledItems = false,
 	installedPlugins
+) {
+	const productList = getProductList(
+		profileItems,
+		includeInstalledItems,
+		installedPlugins
+	);
+	const productIds = productList.map(
+		( product ) => product.id || product.product
+	);
+	return productIds;
+}
+
+/**
+ * Gets the grouped product names and types for items based on the product types and theme selected in the onboarding profiler.
+ *
+ * @param {Object} profileItems Onboarding profile.
+ * @param {Array} installedPlugins Installed plugins.
+ * @return {Array} Objects with grouped product names and types.
+ */
+export function getGroupedOnboardingProducts( profileItems, installedPlugins ) {
```

Comment:
Total nitpick on naming: technically this isn't grouping the products, it's labeling/categorizing them.

Reply:
Thank you for the suggestion, I changed it in the commit e160630